src/components/implementation/no_interface/llbooter/boot_deps.h

```diff
@@ -114,6 +124,96 @@ boot_spd_initaep_get(spdid_t spdid)
 	return cos_sched_aep_get(boot_spd_defcompinfo_get(spdid));
 }
 
+void
+fault_reg_print(spdid_t spdid)
+{
+	struct pt_regs fault_regs;
+
+	struct cos_aep_info *child_aep = boot_spd_initaep_get(spdid);
```

Comment:
Very interesting! It looks like the handler assumes the fault is always in the `INITTHD` of that component?